presto-feature-toggle/README.md

```diff
@@ -0,0 +1,233 @@
+# Feature Toggles
+
+Feature Toggles should allow teams to modify system behavior without changing code. Feature Toggles are configured using Google Guice. The basic definition of toggles is created
+using FeatureToggleBinder. FeatureToggleBinder creates FeatureToggle, and additional configuration can be done using feature configuration.
+
+## Contents
+
+1. [Configuration](#configuration)
+2. [Defining Feature Toggles](#defining-feature-toggles)
+    1. [Simple feature toggle definition](#simple-feature-toggle-definition)
+    2. [Hot reloadable feature toggle definition](#hot-reloadable-feature-toggle-definition)
+    3. [Strategy-based Feature Toggling](#strategy-based-feature-toggling)
+3. [Examples](#examples)
+    1. [Query Cancel Feature](#query-cancel-feature)
+    2. [feature-config.properties file example](#feature-toggle-config-file-example)
+
+## Configuration
+
+To allow feature toggle configuration four lines are needed in the config.properties file
+
+```
+    features.config-source-type=file
+    features.config-source=/etc/feature-config.properties
+    features.config-type=properties
+    features.refresh-period=30s
+```
+
+- `configuration-source-type` is the source type for Feature Toggles configuration
+- `features.config-source` is a source (file) of the configuration
+- `features.config-type` format in which configuration is stored (JSON or properties)
+- `features.refresh-period` configuration refresh period
+
+## Defining Feature Toggles
+
+Feature toggle definition is done in the Google guice module using `FeatureToggleBinder`
+
+## Simple feature toggle definition
+
+```
+    featureToggleBinder(binder)
+        .featureId("featureXX")
+        .bind()
+```
+
+This example creates bindings for `@FeatureToggle("featureXX") Supplier<Boolean> isFeatureXXEnabled.`
+
+```   
+    @Inject
+    public Runner(@FeatureToggle("featureXX") Supplier<Boolean> isFeatureXXEnabled)
+    {
+        this.isFeatureXXEnabled = isFeatureXXEnabled;
+    }
+```
+
+Supplier&lt;Boolean&gt; `isFeatureXXEnabled` can be used to test if the feature is enabled or disabled:
+
+```
+    boolean testFeatureXXEnabled()
+    {
+      return isFeatureXXEnabled.get();
+    }
+```
+
+Switching the feature toggle on/off is done by changing the enabled value from true to false in the configuration source file:
```

Comment:
```suggestion
Switching the feature toggle on or off is done by changing the enabled value from `true` to `false` in the configuration source file:
```
Avoid slashes, see the [Gitlab documentation style guide entry for slashes](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#slashes).